packages/mgt-chat/src/components/Chat/Chat.tsx

```diff
@@ -26,16 +26,23 @@ const useStyles = makeStyles({
     paddingBlockEnd: '12px'
   },
   chatMessages: {
-    paddingInlineStart: '16px',
     height: 'auto',
+    ...shorthands.paddingInline('20px'),
     ...shorthands.overflow('auto'),
     '& img': {
       maxWidth: '100%',
       height: 'auto'
+    },
+
+    '& ul': {
```

Comment:
This rule looks problematic based on the comment from @sebastienlevert

Reply:
True. That's to offset the top level `ul` elements where the messages are listed. I have fixed for the same `ul` elements inside a message by setting it to the initial padding of 40px.